nixos/modules/installer/cd-dvd/installation-cd-graphical-plasma5.nix

```diff
@@ -30,15 +30,27 @@ with lib;
       Version=1.0
       Type=Application
       Name=NixOS Manual
-      Exec=firefox ${config.system.build.manual.manualHTMLIndex}
+      Exec=firefox ${config.system.build.manual.manual}/share/doc/nixos/index.html
```

Comment:
Did the `manualHTMLIndex` variant not work?

Thought on the side: maybe this should be extracted into the nixos manual module…